python/nav/web/netmap/cache.py

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2017 UNINETT AS
+#
+# This file is part of Network Administration Visualized (NAV).
+#
+# NAV is free software: you can redistribute it and/or modify it under
+# the terms of the GNU General Public License version 2 as published by
+# the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.  You should have received a copy of the GNU General Public
+# License along with NAV. If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""Cache utils for NetMap"""
+
+from functools import wraps
+from django.core.cache import cache
+
+# TODO: This cache should be shared by all of NAV?
+# TODO: This cache should be invalidated only when the topology is changed, which
+# is somewhat rare, so set to a reasonable long time for now
+CACHE_TIMEOUT = 60*60
+# Data is collected every 5 minutes by NAV
+TRAFFIC_CACHE_TIMEOUT = 5*60
+
+# Cache model: Index traffic by location, then by traffic layer
+def cache_traffic(layer, f):
```

Comment:
Nitpick here: pylint will spot this as a violation, variable and argument names must have at least three characters (I belive the only exceptions are commonly-accepted short names used in for loops, such as `i` and `j`. This can be read from `.pylintrc`)

(a common guideline for a variable name is that its length should be proportional to its scope)